### ISSUE TITLE
Advice about deploying

### ISSUE THREAD

**Issue by istathar** [honeycombio/kspan#12]
Hello! Do you have any advice about how best to deploy this to a cluster? Should it (for example) be integrated into the **honeycomb-agent**'s Helm chart?

Does it have to be in a particular namespace to see the control plane events?

Presumably it is meant to be a DeamonSet; a sample object of what resources it was given to run would be terrific!

**Comment by mr-ransel** [comment]
I don't believe this would be a DaemonSet, just a standalone pod, with the necessary ServiceAccount, ClusterRole, and ClusterRoleBindings created (although I'm still trying to work out the minimum necessary permissions), because as I understand it kspan is collecting events directly from the API, not the nodes, so only a single instance per cluster is required. Unless you're already running a otel collector service, you'll need to specify an [otlp address](https://github.com/honeycombio/kspan/blob/1000ac6de7f62ecd765416dd865469ce73ff19f5/main.go#L108C42-L108C70) as well. I'm currently working through trying to get this set up myself, so happy to be corrected by someone who's already done it.

**Comment by patrickdung** [comment]
Guys, I created two separate namespaces for kspan and jaeger and is able to use the kspan.

Please refer to [my article](https://www.cloudnative.quest/posts/observability/2023-08-06/using-kspan-and-jaeger-to-visualize-k8s-events-as-spans/) for details.
